src/pages/workspaces/List.js

```diff
@@ -99,8 +99,11 @@ const WorkspaceCard = pure(({ listView, workspace: { workspace: { namespace, nam
   }, [
     div({ style: styles.longTitle }, [name]),
     div({ style: { display: 'flex', alignItems: 'center' } }, [
-      div({ style: styles.longDescription }, [descText]),
-      div({ style: { flex: 'none', width: 400 } }, [lastChanged]),
+      div({ style: styles.longDescription }, [descText]), [
```

Comment:
The syntax here probably isn't what you intended. This adds an array with one element after the description. Arrays are valid, but they're usually for cases when you have a dynamic number of elements. (Also, incidentally, when you use one, you need to provide a `key` prop to each element. If you look at the console, you can see Reach complaining that there's no `key`, which is a hint as to what's happening)